### PR TITLE
[BB2-1179] Node SDK packaging and publish

### PR DESCRIPTION
**Pull request by JFU-GIT**
**JIRA Ticket:**
[BB2-1179](https://jira.cms.gov/browse/BB2-1179)

**User Story or Bug Summary:**

~~Publish the blue button node SDK to npm registry.~~

Some changes are made for packaging and passing api extractor validation.

Provision has been done, npm registry free account (for unscoped public lib) by use name: cms-bluebutton-dev

~~And a soft publishing of cms-bluebutton-sdk 0.1.0 has been done.~~

Per discussion, publishing is revoked (unpublished), in order to test and verify node SDK package and installation in a sample app:

Follow the readme for using the sdk from npm;

Or create local pack tar ball (tgz file) and consume it in your app:

1. Check out PR from repo
2. Run yarn install, yarn build, yarn build:types
3. then do a local packaging : npm pack
4. a tar ball tgz generated at current directory, this tar ball can be consumed (yarn add file:this tar ball file) from the node app you are developing.

### What Should Reviewers Watch For?

If you're reviewing this PR, please check these things, in particular:

- TODO

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (dtisza1) [inline LICENSE:1]: @JFU-GIT This file should be used: https://github.com/CMSgov/cms-open-source-policy/blob/master/LICENSE

**Review by dtisza1 — CHANGES_REQUESTED** [review]
@JFU-GIT This is looking good to me!  

Just one change request related to the license file.

Your testing procedure worked for me to produce the package file.

I was able to install it using:

```
$ yarn add file:cms-bluebutton-sdk-0.1.0.tgz 
```
I haven't tried utilizing the SDK yet, though.

Looks good!

**Review by dtisza1 — APPROVED** [review]
@JFU-GIT This looks good to me!

I'll create a separate PR for the license file and also for the other 3x repos.

Good work on this!

**Review by oragame — COMMENTED** [review]
@JFU-GIT Looks good, I assume the install would not work any more since it was removed from the package repo? Should I try testing anything else before approval?

**Comment by JFU-GIT** [comment]
> @JFU-GIT Looks good, I assume the install would not work any more since it was removed from the package repo? Should I try testing anything else before approval?

Thanks for review and comments, on npm registry - it's unpublished, so not installable from npm.

to verify what can do is the pack and install locally e.g. npm pack -> get the tar ball, go to the location of the app and yarn add file: <path to tar ball>

**Comment by oragame** [comment]
> to verify what can do is the pack and install locally e.g. npm pack -> get the tar ball, go to the location of the app and yarn add file:

OH, I understand now... thanks for clearing that up :)

**Review by oragame — APPROVED** [review]
Looks good! was able to test locally after I realized my local node version was outdated. Nice work!

**Pull request merged**